src/driver/test/CrudActor.yml

```diff
@@ -0,0 +1,24 @@
+SchemaVersion: 2018-07-01
+  
+Actors:
+- Name: CrudActor
+  Type: CrudActor
+  Database: mydb
+  ExecutionStrategy:
+    ThrowOnFailure: true
+  Phases:
+  - Repeat: 1
+    Collection: test
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: { a: 1 }
+        - WriteCommand: updateOne
+          Filter: { a: 1 }
+          Update: { $set: { a: 5 } }
+          Options:
+            WriteConcern:
+              Level: majority
+              TimeoutMillis: 5000
```

Comment:
[lint] Add a newline to the end of this file and address the other whitespace-related errors in CrudActor_test.cpp.

```
$ git diff upstream/master... --check
src/driver/test/CrudActor.yml:2: trailing whitespace.
+  
src/gennylib/test/CrudActor_test.cpp:653: trailing whitespace.
+                  Options: 
src/gennylib/test/CrudActor_test.cpp:691: trailing whitespace.
+                  Options: 
```